lib/crypto.js

```diff
@@ -663,10 +663,10 @@ function filterDuplicates(names) {
 
 // Legacy API
 exports.__defineGetter__('createCredentials',
-  internalUtil.deprecate(function() {
-    return require('tls').createSecureContext;
-  }, 'crypto.createCredentials is deprecated. ' +
-     'Use tls.createSecureContext instead.'));
+                         internalUtil.deprecate(function() {
+                           return require('tls').createSecureContext;
+                         }, 'crypto.createCredentials is deprecated. ' +
+                           'Use tls.createSecureContext instead.'));
```

Comment:
@targos @benjamingr I've rebased, rewritten these lines to hopefully conform more to expectations, and force pushed. Here's what they look like now:

```javascript
// Legacy API
function defineLegacyGetter(legacyName, tlsName) {
  function wrapped() {
    return require('tls')[tlsName];
  }
  const msg = `crypto.${legacyName} is deprecated. Use tls.${tlsName} instead.`;
  exports.__defineGetter__(legacyName, internalUtil.deprecate(wrapped, msg));
}
defineLegacyGetter('createCredentials', 'createSecureContext');
defineLegacyGetter('Credentials', 'SecureContext');
```